src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -5,54 +5,55 @@
 using System;
 using System.Device.Gpio;
 using System.Runtime.InteropServices;
+using NativeLong = System.IntPtr;
 
 internal partial class Interop
```

Comment:
Ideally the contents of this type would be `internal partial class Interop { internal partial class libgpiod { ... } }`, but that could be done later.  (To be different from Interop.libc, etc)

Reply:
I'll take care of this on this PR, at least for libgpiod. The same will be done with libc PInvokes in a subsequent PR.